client/src/main/scala/drt/client/components/UserDashboardPage.scala

```diff
@@ -0,0 +1,33 @@
+package drt.client.components
+
+import drt.client.SPAMain.{Loc, TerminalsDashboardLoc}
+import drt.client.modules.GoogleEventTracker
+import drt.client.services.SPACircuit
+import drt.shared.PortOperatorStaff
+import japgolly.scalajs.react.extra.router.RouterCtl
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.{Callback, ScalaComponent}
+
+object UserDashboardPage {
+
+  case class Props(router: RouterCtl[Loc])
+
+  val component = ScalaComponent.builder[Props]("UserDashboard")
+    .render_P(p => {
+      val loggedInUserRCP = SPACircuit.connect(_.loggedInUserPot)
+      loggedInUserRCP(loggedInUserMP => {
+        <.div(loggedInUserMP().renderReady(user => {
+          if (user.hasRole(PortOperatorStaff))
+            TerminalsExportDashboardPage(user)
```

Comment:
i think it's worth adding a condition for the BorderForceStaff role specifically so we don't risk letting users with neither role access the app

Reply:
so this would only give them access to the TerminalDashboard UI, which would be broken as the endpoints it needs are access restricted. My thinking for this UserDashboardPage is that it will provide a seam to add different types of DashBoards for different types of users. e.g. A planning user might want a forecast summary view here, etc.